src/pages/create/EventForm.tsx

```diff
@@ -0,0 +1,70 @@
+import { yupResolver } from '@hookform/resolvers/yup';
+import { useMemo } from 'react';
+import { useForm } from 'react-hook-form';
+import { useTranslation } from 'react-i18next';
+import * as yup from 'yup';
+
+import type { Calendar } from '@/hooks/api/events';
+import type { StepsConfiguration } from '@/pages/Steps';
+import { Steps } from '@/pages/Steps';
+import { Page } from '@/ui/Page';
+
+import { CalendarStep } from './CalendarStep';
+import { ThemeStep } from './ThemeStep';
+import { TypeStep } from './TypeStep';
+
+type EventType = 'event' | 'place';
```

Comment:
```suggestion
type OfferType = 'event' | 'place';
```

Also the file should better be called `OfferForm`, since this is the correct term we use for something that is either a event or place

Reply:
It's still in draft, I'm changing a lot of things as we speak